Adding 404 error info (#4125)
segmentio/segment-docs: src/connections/destinations/catalog/actions-fullstory-cloud/index.md

> info ""
> Events that you send through to FullStory through a Cloud-mode connection count towards your FullStory server event quota. To see your company’s current quota allotment, view the Subscription information on the Account Settings page in FullStory.

## Troubleshooting

### Why am I getting a ‘404 Not Found’ error?

> The user for which the API request is being made can not be found in the identified set of users within your FullStory organization. If you expect that user to already exist, please confirm by searching for that “User ID” within the FullStory application. Also, double check that you are using an API key from the same organization.

Reviewer: ```suggestion
The user for which the API request is being made can not be found in the identified set of users within your FullStory organization. If you expect that user to already exist, you can search for that User ID in FullStory to confirm. Also, double check that you are using an API key from the same organization.
```

> Data being sent in server-side for users must match an already existing userId that was sent in via a client-side connection.

Reviewer: ```suggestion
Data sent server-side for users must match an already existing userId that was sent from a client-side connection.
```
